src/style.css

```
@@ -0,0 +1,102 @@
+
+@font-face {
```

Don't leave a redundant blank lines
```suggestion
@font-face {
```

---

Redundant line has been removed